crates/ark/src/variables/variable.rs

```diff
@@ -106,6 +113,35 @@ impl WorkspaceVariableDisplayValue {
         Self::new(String::from(""), false)
     }
 
+    fn from_language(value: SEXP) -> Self {
+        if r_inherits(value, "formula") {
+            return Self::from_formula(value);
+        }
+
+        return Self::from_error(Error::Anyhow(anyhow!("Unexpected language object type")));
+    }
+
+    fn from_formula(value: SEXP) -> Self {
+        let display_value: Result<RObject, Error> = RFunction::new("base", "format")
```

Comment:
`~{}` will be multiline and return a character vector of length > 1. I think we could use `as.character(x)` and if not length 1 elide with `...`? In that case we'd mark it truncated?

Reply:
`as.character()` adds a some newline separators which might not be necessary. Still using `format` and then trimming/truncating when necessary.